packages/buidler-core/src/internal/cli/cli-with-a-typo.ts

```diff
@@ -2,6 +2,6 @@
 /* tslint:disable */
 import chalk from "chalk";
 
-console.error(chalk.cyan("You probably meant to type buidler. We got you."));
+console.error(chalk.cyan("You probably meant to type hardhat. We got you."));
```

Comment:
We can delete this file. It should also be removed from the `bin` section of `package.json`.